hazelcast/src/test/java/com/hazelcast/json/internal/JsonSchemaHelperNullTest.java

```diff
@@ -140,11 +140,6 @@ public void test_givenWrongPattern_returnNull_whenObject_queriedByExtraIndex() t
 
     // tests on nested object
 
-    @Test
-    public void testNestedObject_givenInvalidPath_shouldCreateNullPattern_whenObject_pathMissingAttribute() throws IOException {
```

Comment:
This test ensured that `JsonSchemaHelper#createPattern` returned null pattern when a path to non-terminal value was queried. This is changed now. createPattern is allowed to create patterns for non-terminal values.